table.go

```diff
@@ -242,13 +242,19 @@ func newTable(
 		Name: "index_size",
 		Help: "Number of granules in the table index currently.",
 	}, func() float64 {
+		if t.ActiveBlock() == nil {
```

Comment:
Wouldn't we want to save the result of `t.ActiveBlock` to a variable and test it for nil? Otherwise by the below call to `t.ActiveBlock` might be a different value again, no?

Reply:
it's only nil during initialization
so it can't go backwards to nil
but it will hedge against that in the future so wil change